images/kubekins-e2e-v2/cloudbuild.yaml

```diff
@@ -5,7 +5,7 @@ steps:
     - build
     - --tag=us-central1-docker.pkg.dev/$PROJECT_ID/images/kubekins-e2e:$_GIT_TAG-$_CONFIG
     - --build-arg=IMAGE_ARG=us-central1-docker.pkg.dev/$PROJECT_ID/images/kubekins-e2e:$_GIT_TAG-$_CONFIG
-    - --platform=linux/amd64,linux/arm64
+    - --platform=linux/amd64,linux/arm64,linux/ppc64le
```

Comment:
Can you add these lines at the end of the file?

```
options:
  machineType: E2_HIGHCPU_32
```

Reply:
@upodroid I already see this at the end of `cloudbuild.yaml` https://github.com/kubernetes/test-infra/blob/master/images/kubekins-e2e-v2/cloudbuild.yaml#L39
Am I missing something?